src/TSTransformer/util/jsx/getJsxIndex.ts

```diff
@@ -0,0 +1,18 @@
+import luau from "@roblox-ts/luau-ast";
+import assert from "assert";
+import { TransformState } from "TSTransformer/classes/TransformState";
+
+function getJsxIndex(factory: string) {
+	const [namespace, index] = factory.split(".");
+	return luau.property(luau.id(namespace), index);
+}
+
+export function getCreateFragmentIndex(state: TransformState) {
+	assert(state.compilerOptions.jsxFragmentFactory);
+	return getJsxIndex(state.compilerOptions.jsxFragmentFactory);
+}
```

Comment:
What if the jsxFragmentFactory is not set at all. will TS error on that?
If so, is it reasonable to still support this for the case of ts-ignore?
Or if not, is an assert failure the best way to go about failing?

Same questions for regular jsxFactory